bcos-boostssl/websocket/WsService.h

```diff
@@ -182,6 +187,12 @@ class WsService : public std::enable_shared_from_this<WsService>
         m_handshakeHandlers.push_back(_handshakeHandler);
     }
 
+    void setReconnectedPeers(EndPointsPtr _reconnectedPeers)
+    {
+        m_reconnectedPeers = _reconnectedPeers;
+    }
+    EndPointsPtr reconnectedPeers() const { return m_reconnectedPeers; }
```

Comment:
Same as above, add lock here to ensure thread-safe.

Reply:
has added a lock to ensure thread-safe.